lib/housekeeping.js

```diff
@@ -38,17 +38,109 @@ class HouseKeeper {
     this.tagger = tagger;
   }
 
-  async _handleVolumeData(volume) {}
+  _calculateVolumeTotals(volumes) {}
+
+  _submitVolumeTotals(totalsByCategory) {
+    // Assumption:
+    // totalsByCategory is organized as follows:
+    // { 
+    //   ${region}: [ 
+    //     {
+    //       ${vol-type}: {
+    //         active: {
+    //           gb: value,
+    //           count: value
+    //         },
+    //         unused: {
+    //           gb: value,
+    //           count: value
+    //         }
+    //       }
+    //     },
+    //     ...
+    //   ]
+    // }
+
+    //assert(typeof totalsByCategory === 'object');
+
+    //assert(Object.keys(totalsByCategory).length === 1);
+    let region =  Object.keys(totalsByCategory)[0];
+    
+    let volumeTypes = totalsByCategory[region];
+    //assert(typeof volumeTypes === 'object');
```

Comment:
I assume these `assert`s are leftover debugging things? We can remove them before this is merged. If they're useful to keep around feel free to leave the asserts in, but no need to comment them out in that case!

Reply:
That's something I wanted to ask about, actually. I commented them out for now primarily so the PR would pass the tests. I was thinking that it made sense to include these assert statements as a way to guarantee that the right data is submitted to the statsum series, but does it make sense to include these past the testing phase?